pkg/host-disk/host-disk.go

```diff
@@ -99,11 +99,6 @@ func ReplacePVCByHostDisk(vmi *v1.VirtualMachineInstance, clientset kubecli.Kube
 			}
 			// PersistenVolumeClaim is replaced by HostDisk
 			volumeSource.PersistentVolumeClaim = nil
-			// Set ownership of the disk.img to qemu
-			if err := ephemeraldiskutils.DefaultOwnershipManager.SetFileOwnership(file); err != nil && !os.IsNotExist(err) {
-				log.Log.Reason(err).Errorf("Couldn't set Ownership on %s: %v", file, err)
-				return err
-			}
```

Comment:
I thought this was already removed by that refactor of the host-disk package you did recently, right?!

Reply:
Rebased so github should be not confused anymore here.